### PR TITLE
Update drivers doc for vSphere CSI driver 2.5 GA

### PR DESCRIPTION
**Pull request by lintongj**
Update drivers doc for vSphere CSI driver 2.5 GA

### CODE REVIEW

**Comment by lintongj** [comment]
@xing-yang PTAL

File: book/src/drivers.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
...
 [Veritas InfoScale Volumes](https://www.veritas.com/solution/virtualization/containers.html) | `org.veritas.infoscale` | v1.2 | A Container Storage Interface (CSI) Driver for Veritas InfoScale volumes | Persistent | Read/Write Multiple Pods | Yes | Snapshot, Expansion, Cloning
-[vSphere](https://github.com/kubernetes-sigs/vsphere-csi-driver) | `csi.vsphere.vmware.com` | v1.0 | A Container Storage Interface (CSI) Driver for VMware vSphere | Persistent | Read/Write Single Pod (Block Volume) <br/><br/> Read/Write Multiple Pods (File Volume) | Yes | Raw Block,<br/><br/>Expansion (Block Volume),<br/><br/>Topology Aware (Block Volume)
+[vSphere](https://github.com/kubernetes-sigs/vsphere-csi-driver) | `csi.vsphere.vmware.com` | v1.4 | A Container Storage Interface (CSI) Driver for VMware vSphere | Persistent | Read/Write Single Pod (Block Volume) <br/><br/> Read/Write Multiple Pods (File Volume) | Yes | Raw Block,<br/><br/>Expansion (Block Volume),<br/><br/>Topology Aware (Block Volume)<br/><br/>Snapshot (Block Volume)
```
Comment (xing-yang) [inline book/src/drivers.md:120]: Can you add a "," after "Topology Aware (Block Volume)"?

**Pull request merged**